ios/PurchasesHybridCommon/PurchasesHybridCommon/CommonFunctionality.swift

```diff
@@ -109,25 +98,25 @@ import Purchases
     @objc(restoreTransactionsWithCompletionBlock:)
     static func restoreTransactions(completion: @escaping ([String: Any]?, ErrorContainer?) -> Void) {
         let purchaserInfoCompletion = purchaserInfoCompletionBlock(from: completion)
-        Purchases.shared.restoreTransactions(purchaserInfoCompletion)
+        Purchases.shared.restorePurchases(completion: purchaserInfoCompletion)
```

Comment:
is it worth renaming the local vars here? to `customerInfoCompletion`?

Reply:
I should have added a note, I have the rename [here](https://github.com/RevenueCat/purchases-hybrid-common/pull/131/files#diff-463b908f62c781c1e5d798d18af1016da664f05a238692e5181c0044e15ee2f9R98), 2 PRs after this one